tests/unit/backend/wmg/api/test_v2.py

```diff
@@ -756,14 +765,16 @@ def test__filter_request_with_filter_dims__returns_valid_filter_dims__base_case(
                 "sex_terms": [],
                 "tissue_terms": [{"tissue_ontology_term_id_0": "tissue_ontology_term_id_0_label"}],
             }
+
             self.assertEqual(json.loads(response.data)["filter_dims"], expected_filters)
 
     @patch("backend.wmg.api.v2.fetch_datasets_metadata")
     @patch("backend.wmg.api.v2.gene_term_label")
+    @patch("backend.wmg.api.v2.ethnicity_term_label")
     @patch("backend.wmg.api.v2.ontology_term_label")
     @patch("backend.wmg.api.v2.load_snapshot")
     def test__filter_request_with_filter_dims__returns_valid_filter_dims(
-        self, load_snapshot, ontology_term_label, gene_term_label, fetch_datasets_metadata
+        self, load_snapshot, ontology_term_label, ethnicity_term_label, gene_term_label, fetch_datasets_metadata
```

Comment:
@atarashansky - Should a similar test case including comma separated `self_reported_ethnicity_ontology_term_id` be added here too?